src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -0,0 +1,72 @@
+package com.bootcamp.demo.model;
+
+/**
+ * An User object stores first and last name, email, password, phoneNumber and gender of an user
+ *
+ * Next needed update: to find a way to store encrypted password, not the real one
+ *
+ * @author Denisa Dragota
+ * @version 11/11/2021
+ */
+
+public class User {
+
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String password;
+    private String phoneNumber;
+    private String gender;
+
+    public User(){};
```

Comment:
Remove the unnecessary semicolon after the constructor. 
Also, if you have only the default constructor, without any implementation or without adding another constructor with parameters, you can omit it.